src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskInitiateByIdControllerTest.java

```diff
@@ -124,7 +124,6 @@ public void should_return_a_201_when_initiating_a_process_application_task_by_id
             getTaskResource(result, "task-supervisor"),
             Map.ofEntries(
                 entry("read", true),
-                entry("refer", true),
                 entry("own", false),
                 entry("manage", true),
                 entry("execute", false),
```

Comment:
What about Assign, Unassign and Complete?

Reply:
From base branch 1528